examples/basic_bitcoin/src/backend/index.ts

```diff
@@ -0,0 +1,116 @@
+import { jsonStringify } from 'azle';
+import { BitcoinNetwork } from 'azle/canisters/management';
+import express, { Request } from 'express';
+
+import * as bitcoinApi from './bitcoin_api';
+import * as bitcoinWallet from './bitcoin_wallet';
+import { SendRequest } from './types';
+
+// The bitcoin network to connect to.
+//
+// When developing locally this should be `Regtest`.
+// When deploying to the IC this should be `Testnet`.
+// `Mainnet` is currently unsupported.
+const NETWORK: BitcoinNetwork = determineNetwork(
+    process.env.BITCOIN_NETWORK
+) ?? {
+    testnet: null
+};
+
+// The derivation path to use for ECDSA secp256k1.
+const DERIVATION_PATH: Uint8Array[] = [];
+
+// The ECDSA key name.
+const KEY_NAME: string = determineKeyName(NETWORK);
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
```

Comment:
because I like having a / endpoint that does something and it helped me make sure all of our stuff was getting set up, we can get rid of it if you like

Reply:
done